plugins/env/app/models/environment_variable.rb

```diff
@@ -67,6 +67,12 @@ def env_vars_from_db(deploy, deploy_group, **args)
     end
 
     def env_vars_from_external_groups(project, deploy_group)
+      external_groups = project.external_environment_variable_groups
+
+      external_groups.concat(project.environment_variable_groups.select(&:external_url?).map do |env_group|
+        ExternalEnvironmentVariableGroup.new(url: env_group.external_url)
+      end)
+
       project.external_environment_variable_groups.each_with_object({}) do |group, envs|
```

Comment:
it should be `external_groups` here?

Reply:
yes, thx!

Reply:
... means I'm missing a test too

Reply:
hehe, it actually worked because it modified the external_environment_variable_groups ... 2 bugs make 1 feature ;)